src/components/highlighter/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Prism from 'prismjs';
+import Highlight from 'prism-react-renderer';
+import { Box, Flex, useTheme } from '@blockstack/ui';
+
+import { Token, GetTokenProps, RenderProps } from '@components/highlighter/types';
+import { theme } from '@components/highlighter/prism-theme';
+
+const lineNumberWidth = 60;
+const getLineNumber = (n: number) => (n + 1 < 10 ? '0' : '') + (n + 1);
```

Comment:
This should probs take a total length right, so padding can be 2 0's if it's 100+ long?